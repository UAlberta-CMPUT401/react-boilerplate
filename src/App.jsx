import React from 'react';

function App() {
  return (
    <div className="App">
      <h1>Hello Jashwanth!</h1>
    </div>
  );
}

export default App;
