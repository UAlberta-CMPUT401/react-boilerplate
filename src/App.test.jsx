import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

test('Check App renders or not', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
