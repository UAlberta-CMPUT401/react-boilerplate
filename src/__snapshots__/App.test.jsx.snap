// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check App renders or not 1`] = `
<div
  className="App"
>
  <h1>
    Hello World Jashwanth!
  </h1>
</div>
`;
